lib/rust/ensogl/component/grid-view/src/entry.rs

```diff
@@ -76,6 +76,7 @@ ensogl_core::define_endpoints_2! { <Model: (frp::node::Data), Params: (frp::node
         set_size(Vector2),
         set_params(Params),
         set_location((Row, Col)),
+        set_position(Vector2),
```

Comment:
`set_position` suggests that the Entry implementation should take care of setting the position in display object, but actually it is set by Grid view. So, maybe `position_set`?